bciers/apps/registration/app/components/operations/OperationsPage.tsx

```diff
@@ -21,6 +22,7 @@ export default async function OperationsPage({
    * getServerSession requires passing the same object you would pass to NextAuth
    */
   const session = await getServerSession(authOptions);
+  // @ts-ignore
```

Comment:
I ~think the auth PR should fix the session types. If it's not in before this PR, I'll add a bullet to its card to remove this ignore